docs/API.md

```diff
@@ -1808,6 +1808,89 @@ The Meta field is populated according to the include flag  - see the `meta`
 path for more info on how to use this.
 The `limit` flag is the same as for the "search" path.
 
+### List
+
+#### GET list
+
+The `list` path lists within the latest version of charms and bundles
```

Comment:
why just the latest version? I think it should list all of them (at least with an option)